nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-client-service/pom.xml

```diff
@@ -150,6 +172,7 @@
                             <httpPort>9400</httpPort>
                             <version>5.6.2</version>
                             <timeout>90</timeout>
+                            <pathInitScript>src/test/resources/setup.script</pathInitScript>
```

Comment:
I don't think this works if you run -Pintegration-tests from the bundle level, but it does from the client-service level

Reply:
Verified that's the case. Going to have to look into that.

Reply:
Resolved.